server/routes/business.js

```diff
@@ -54,10 +54,12 @@ router.post('/create', async (req, res) => {
     //Check if owner already has a business
     let ownerHasBusiness = await Business.findOne({ownerId: req.body.ownerId}).exec();
```

Comment:
We should avoid this: Prefer using top-down structure, searching for the business' flag in an Owner entry.